### ISSUE TITLE
Error in check_ploidyOgMatch

### ISSUE THREAD

**Issue by adam-ryan-97** [jtlovell/GENESPACE#46]
Hi - I'm using the 1.0.0 dev version - it worked great on a smaller set of my genomes. I'm getting the following error now that I've expanded the set:

Error in if (tmp$pHOG > tmp$pOG) { :
  missing value where TRUE/FALSE needed
Calls: run_genespace -> annotate_bed -> check_ploidyOgMatch
Execution halted

**Comment by jtlovell** [comment]
Hi Adam, thanks for trying out v1!
Can you ping me via email and we can get this bug figured out? jlovell[at]hudsonalpha[dot]org

**Comment by jtlovell** [comment]
OK, there were quite a few issues here. Enough that I updated to v1.0.2 (still at /dev). 
Updates:

- there were certain situations where detecting HOG/OGs was not straightforward and caused it to error out. Now, setting `useHOGs = NULL` is no longer allowed and must be either T/F. Default is `init_genespace(..., useHOGs = TRUE)`. 
- when a genome is provided that is not at chromosome scale, plotting can become difficult. Here, one of your genomes had good syntenic blocks on 179 "chromosomes". This means that there were >32k facets to plot, which should be doable, but is incredibly slow. To deal with this, I have added an argument: `init_genespace(..., dotplots = c("check", "always", "never"))`. The previous version was equivalent to `dotplots = "always"`. Now, by default (dotplots = "check"), genespace will write plots as long as there are less than 10k facets. The "never" option is a good choice if you have lots of genomes and want to optimize speed.
